FogCreek/Glitch-Community
# Fuschia canoe: Filter out "My Stuff" from "More by" and "Included in" sections

## Links
* https://fuschia-canoe.glitch.me
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/244

## Changes:
* removes "My Stuff" collections from the "More by" and "Included in" sections on the project and collection pages. 

## How To Test:
* include a project in a "My Stuff" collection, you should never see that collection show up on the project page under the Included In section, similarly when you go to your collection pages you should never see the "My Stuff" collection show up under the "More by" section
